device-types/Netgate/8200-max.yaml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: Netgate
+model: 8200-max
```

Comment:
It looks like the proper name of this is `8200 Max PFSense+ Security Gateway`

Reply:
addressed in [58623ef](https://github.com/netbox-community/devicetype-library/pull/1709/commits/58623ef29ea86f1fc374c6f6ace75ceacd013f7a)